src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1952,15 +1958,26 @@ def delete_cc_resource(
     resource_group_name: str,
     cluster_name: str,
     no_wait: bool,
+    force: bool = False,
 ) -> LROPoller[None]:
     print(f"Step: {utils.get_utctimestring()}: Deleting ARM resource")
     try:
-        poller: LROPoller[None] = sdk_no_wait(
-            no_wait,
-            client.begin_delete,
-            resource_group_name=resource_group_name,
-            cluster_name=cluster_name,
-        )
+        poller: LROPoller[None]
+        if force:
```

Comment:
how about reformatting sdk_no_wait call to avoid duplicate statements by declaring parameters earlier:
params = {"force": True} if force else {}
poller = sdk_no_wait(
    no_wait,
    client.begin_delete,
    resource_group_name=resource_group_name,
    cluster_name=cluster_name,
    params=params,
)

Reply:
just trying to avoid touching the exisitng path which is working. I am not sure if empty params is handled on sdk